documentation/IDTA-01001/modules/ROOT/pages/Spec/IDTA-01001_Metamodel_DataTypes.adoc

```diff
@@ -12,24 +12,39 @@ SPDX-License-Identifier: CC-BY-4.0
 
 == Predefined Simple Data Types
 
-The metamodel of the Asset Administration Shell uses basic data types as defined in the link:https://www.w3.org/XML/Core/[XML Schema Definition] (XSD)footnote:[https://www.w3.org/XML/Core/, former https://www.w3.org/XML/Schema]. See Table 7 for an overview of the used types. Their definition is outside the scope of this document.
+The metamodel of the Asset Administration Shell uses some of the predefined simple data types of the  link:https://www.w3.org/XML/Core/[XML Schema Definition] (XSD) 
+as its basic data types footnote:[https://www.w3.org/XML/Core/, former https://www.w3.org/XML/Schema]. See Table 7 for an overview of the used types. Their definition is outside the scope of this document.
 
-The meaning and format of xsd types is specified in https://www.w3.org/XML/Schema. The simple type "langString" is specified in the Resource Description Framework (RDF)footnote:[see: https://www.w3.org/TR/rdf11-concepts/].
+The meaning and format of xsd types is specified in https://www.w3.org/XML/Schema. The simple type "langString" is specified in the Resource Description Framework (RDF) footnote:[see: https://www.w3.org/TR/rdf11-concepts/].
 
 See Clause 5.3.12.6 for constraints on types.
 
 .Simple Data Types Used in Metamodel
 [cols="10%,19%,38%,33%",options="header",]
 |===
 |Source |Basic Data Type |Value Range |Sample Values
+
 |xsd |string |Character string (but not all Unicode character strings) |"Hello world", "Καλημέρα κόσμε", +
 "ハローワールド""
+
 |xsd |base64Binary |base64-encoded binary data |"a3Vtb3dhc2hlcmU="
+
 |xsd |boolean |true, false |true, false
+
+|xsd |anyUri | Represents an Internationalized Resource Identifier Reference (IRI) conformant to https://www.ietf.org/rfc/rfc3987.txt[RFC 3987], can be absolute or relative, and may have an optional fragment identifier  a| {blank}./Specification.pdf
```

Comment:
This is inconsistent to line 203 - where we write:

...as URIs per [RFC 3986] and its successor(s).

Please replace with:
|xsd |anyUri | Represents an Uniform Resource Identifier (URI) conformant to https://www.ietf.org/rfc/rfc3986[RFC 3986] and its successor(s), can be absolute or relative, and may have an optional fragment identifier  a| {blank}./Specification.pdf

Reply:
You are right. Resolved with https://github.com/admin-shell-io/aas-specs/pull/358/commits/d59781308ee906a9e37533adeef4b1408ff65c9f